pennylane_lightning/core/src/simulators/lightning_kokkos/catalyst/CMakeLists.txt

```diff
@@ -0,0 +1,92 @@
+cmake_minimum_required(VERSION 3.20)
+
+project(lightning_kokkos_catalyst LANGUAGES CXX)
+
+set(CATALYST_FILES LightningKokkosSimulator.cpp CACHE INTERNAL "" FORCE)
```

Comment:
We always do like that, so I decided to follow our standard.

Reply:
Maybe @mlxd can offer some explanation for that.

Reply:
To clarify the use of the FORCE keyword here serves to overwrite existing entries.

Reply:
If the cache was already populated (e.g. you ran a cmake configuration previously, then made an update, or just mirrored the name elsewhere) the previous cache may be used. Force ensures it is always up to date with the current files before proceeding.

It may be fine to remove it in this case, but it's no harm, and common practice in many docs on this.

Reply:
Though, naming this `LK_CATALYST_FILES` would probably be safer, to avoid aliasing from external packages using the name.

Reply:
I agree, thanks for the suggestion.